frontend/src/components/browse-sightings-page/BrowseSightings.scss

```
@@ -14,19 +14,27 @@
     align-self: center;
   }
 
-  h3 {
+  .sighting-card_title {
```

Very close! BEM dictates two underscores:
```suggestion
  .sighting-card__title {
```

---

ok thanks, done!